src/preface/abstract.adoc

```diff
@@ -1,3 +1,5 @@
+[discrete]
+// [discrete] macro hides headline from TOC.
```

Comment:
Would be awesome to have those `// [discrete] macro hides headline from TOC.` at the sime line or before `[discrete]` rather than after, if possible. I imagine `[discrete] // hides from TOC` if that works?